### ISSUE TITLE
Author biographies should be a list instead of sections/headings

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#6901]
An accessibility audit recommended to convert the author biographies on the landing page to a list without headings. So instead of:

```
<section>
  <h3>Author Name</h3>
  <p>...</p>
</section>
```

They should be:

```
<ul>
  <li>
   <div>Author Name</div>
   <p>...</p>
  </li>.
  <li>
    ...
  </li>
</ul>
```

**Comment by NateWr** [comment]
Merged to `main`. Thanks @israelcefrin!
